.github/workflows/deploy_site.yml

```diff
@@ -0,0 +1,76 @@
+name: Deploy site
+
+on:
+  push:
+    branches:
+    - master
+  workflow_dispatch:
+
+jobs:
+  render-pdf:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    
+    # Action for collection of short branchname in commit mesage
+    - name: Inject short variables
+      uses: rlespinasse/github-slug-action@v3.x
+
+    - name: Install python packages
+      run: |
+          sudo apt update 
+          wget --version 
+          pip install mkdocs
+          pip install mkdocs-cinder
+          pip install markdown-include
+          sudo apt-get -y install xfonts-encodings libxrender-dev libfontconfig-dev libxext-dev
+          wget https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.focal_amd64.deb
+          sudo apt -y install ./wkhtmltox_0.12.6-1.focal_amd64.deb
+          wkhtmltopdf --version
+          mkdocs --version
+
+    - name: Initialize gh-pages checkout
+      run: |
+          mkdir -p site
+          cd site
```

Comment:
Can you not remove the `cd site` here? The next command does not make sense in that directory, also, the name of the step does not match the content anymore.

_Maybe remove the whole step?_